template/src/lib/ui/widgets/HistogramWidgetUI.js

```diff
@@ -205,6 +223,7 @@ function HistogramWidgetUI(props) {
     tooltipFormatter,
     xAxisFormatter,
     yAxisFormatter,
+    height = 174,
```

Comment:
That's accurate! Maybe better with spacing?

Reply:
it's the default size asked by design. I can change to spacing but I think that is not necessary

Reply:
We should define default sizes respecting the grid, so I'd better go with 176 -> spacing(22).
Also we need to define a default widget size so they can be placed one next to each other and look good. Does it make sense for you?
/cc @javigil-geo @alasarr

Reply:
fixed